### PR TITLE
Remove required request body

### PR DESCRIPTION
**Pull request by fbjorn**
- Remove validation of Mandatory request body
- Add validation for authorization headers

### CODE REVIEW

File: openapi_to_fastapi/tests/data/ihan/CompanyBasicInfo.json [hunk 1]
```diff
@@ -42,7 +42,25 @@
               }
             }
           }
-        }
+        },
+        "parameters": [
```
Comment (lietu) [inline openapi_to_fastapi/tests/data/ihan/CompanyBasicInfo.json:46]: Is there any special way to mark these as optional?

File: openapi_to_fastapi/tests/data/ihan/WeatherCurrentMetric.json [hunk 2]
```diff
@@ -22,6 +22,22 @@
             },
             "name": "x-signature",
             "in": "header"
+          },
+          {
```
Comment (lietu) [inline openapi_to_fastapi/tests/data/ihan/WeatherCurrentMetric.json:26]: And these.

File: openapi_to_fastapi/tests/data/ihan/CompanyBasicInfo.json (hunk 1 above)
Comment (fbjorn) [reply openapi_to_fastapi/tests/data/ihan/CompanyBasicInfo.json:46]: if there's no `"required": true` set, then they're considered optional

File: openapi_to_fastapi/validator/ihan_standards.py [hunk 3]
```diff
@@ -82,17 +90,24 @@ def validate_spec(spec: dict):
     if not component_schemas:
         raise SchemaMissing('No "components/schemas" section defined')
 
-    if not post_route.get("requestBody"):
-        raise RequestBodyMissing
-    if not post_route["requestBody"].get("content"):
-        raise RequestBodyMissing
-    validate_component_schema(post_route["requestBody"], component_schemas)
+    if post_route.get("requestBody") and post_route["requestBody"].get("content"):
```
Comment (joakimnordling) [inline openapi_to_fastapi/validator/ihan_standards.py:93]: Could this be simplified to just `post_route.get("requestBody", {}).get("content"):`?

Comment (fbjorn) [reply openapi_to_fastapi/validator/ihan_standards.py]: true.. missed it somehow

**Review by joakimnordling — APPROVED** [review]

**Pull request merged**